README.md

```diff
@@ -79,7 +79,7 @@ pinpoint| 1.8.0-RC1
         ```
```

Comment:
1. "1. export COLLECTOR_CONFIG=/full path of collector.conf/2. run  collector-agent" should be step 5 and step 6.
2. "Log_Level=ERROR # debug in dev" will get error.